frontend/public/components/cluster-settings/ldap-idp-form.tsx

```diff
@@ -249,46 +256,56 @@ export class AddLDAPPage extends PromiseComponent<{}, AddLDAPPageState> {
               aria-describedby="bind-password-help"
             />
             <div className="help-block" id="bind-password-help">
-              Password to bind with during the search phase.
+              {t('ldap-idp-form~Password to bind with during the search phase.')}
             </div>
           </div>
           <div className="co-form-section__separator" />
-          <h3>Attributes</h3>
-          <p className="co-help-text">Attributes map LDAP attributes to identities.</p>
+          <h3>{t('ldap-idp-form~Attributes')}</h3>
+          <p className="co-help-text">
+            {t('ldap-idp-form~Attributes map LDAP attributes to identities.')}
+          </p>
           <ListInput
-            label="ID"
+            label={t('ldap-idp-form~ID')}
             required
             initialValues={attributesID}
             onChange={this.attributesIDChanged}
-            helpText="The list of attributes whose values should be used as the user ID."
+            helpText={t(
+              'ldap-idp-form~The list of attributes whose values should be used as the user ID.',
+            )}
           />
           <ListInput
-            label="Preferred Username"
+            label={t('ldap-idp-form~Preferred username')}
             initialValues={attributesPreferredUsername}
             onChange={this.attributesPreferredUsernameChanged}
-            helpText="The list of attributes whose values should be used as the preferred username."
+            helpText={t(
+              'ldap-idp-form~The list of attributes whose values should be used as the preferred username.',
```

Comment:
In docs, we use "user name" vs "username" per IBM style guidelines. Probably out of scope for this PR, but just a heads-up in case we want to sync up our usage on that.